conf/messages

```diff
@@ -540,34 +540,34 @@ status.feepaid = Fee paid
 status.underreview = Decision pending
 status.decisionmade = Decision made
 status.incomplete.heading = Application incomplete
-status.incomplete.description = You’ve started the application form but you haven’t finished it. You have 28 days to finish the application, from when you last saved your progress.
-status.incomplete.buttontext = Return to saved application
+status.incomplete.description = You have started your application but have not finished it. You need to complete all sections on the about your business page.
+status.incomplete.description1 = You have 28 days to finish the application from when you last saved your progress.
+status.incomplete.buttontext = Return to your application
 status.submissionready.heading = Application not submitted
-status.submissionready.description = You’ve completed all sections of the form, but you haven’t submitted it or paid the fee.
-status.submissionready.description2 = Your application won’t be reviewed until the fee has been paid.
-status.submissionready.description3 = You have 28 days to submit the application and pay the fee, from when you last saved your progress.
-status.hassomethingchanged = Has something changed?
-status.submissionready.changelink1 = edit your application
-status.submissionready.buttontext = Submit application
+status.submissionready.description = You have completed your application but have not submitted it.
+status.submissionready.description2 = You have 28 days to submit your application from when you last saved your progress.
+status.submissionready.description3 = To submit your application you need to complete a declaration, and pay the fees. HMRC will then review your application.
+status.hassomethingchanged = About your business
+status.submissionready.changelink1 = Check or update your business information
+status.submissionready.buttontext = Continue
 status.submissionreadyforreview.heading = Application pending
-status.submissionreadyforreview.description = HMRC will review your application after you have paid the relevant fees. Fees will need to be paid within 28 days. If you have not paid yet you can
+status.submissionreadyforreview.description = You have submitted your application, and have 28 days to pay the fees. If you have not already paid, 
```

Comment:
Should this line be ending with a comma?

Reply:
this is one part of a longer statement that has some dynamic content based on state. so yes the comma is valid here